app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -41,10 +41,18 @@
           <% elsif @step_by_step_page.can_be_published? %>
             <p class="govuk-body">Before publishing check for broken links.</p>
             <p class="govuk-body">After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
-            <%= render "govuk_publishing_components/components/button", {
-              text: "Publish changes",
-              href: step_by_step_page_publish_path(@step_by_step_page)
-            } %>
+            <% if @step_by_step_page.has_been_published? %>
+              <%= render "govuk_publishing_components/components/button", {
+                text: "Publish changes",
+                href: step_by_step_page_publish_path(@step_by_step_page)
+              } %>
+            <% else %>
+              <%= form_for(@step_by_step_page, url: step_by_step_page_publish_path(@step_by_step_page), method: :post) do %>
+                <input type="hidden" name="update_type" value="minor">
+                <input type="hidden" name="change_note" value="">
```

Comment:
Rather than adding hidden fields here, I think it would be better for the `publish` action in the controller to set the default values for `PublishIntent`.